tests/conftest.py

```diff
@@ -256,6 +256,31 @@ def HELIOTRON_vac2(tmpdir_factory):
     return HELIOTRON_vacuum2_out
 
 
+@pytest.fixture(scope="session")
+def ESTELL_older(tmpdir_factory):
+    """Run ESTELL_older example."""
+    input_path = ".//tests//inputs//ESTELL_older"
+    output_dir = tmpdir_factory.mktemp("result")
+    desc_h5_path = output_dir.join("ESTELL_older.h5")
+
+    cwd = os.path.dirname(__file__)
+    exec_dir = os.path.join(cwd, "..")
+    input_filename = os.path.join(exec_dir, input_path)
+
+    print("Running ESTELL_older test.")
+    print("exec_dir=", exec_dir)
+    print("cwd=", cwd)
+
+    args = ["-o", str(desc_h5_path), input_filename, "-vv"]
+    main(args)
```

Comment:
this is re-running the current version of desc, is that what you want?

Reply:
This whole function has been removed from conftest.py